src/AppInstallerCLITests/TestHandler.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "TestCommon.h"
+#include <Rest/Schema/1_0/Interface.h>
+#include <Rest/Schema/IRestClient.h>
+#include "TestHandler.h"
+
+std::shared_ptr<TestHandler> GetTestHandler(

```

Comment:
I think this should be under TestCommon namespace? Also applies to the TestHandler.h